http/src/main/resources/init-resources/startup.sh

```diff
@@ -95,6 +95,13 @@ else
     GSUTIL_CMD='gsutil'
     GCLOUD_CMD='gcloud'
     DOCKER_COMPOSE='docker-compose'
+
+    # Update old welder docker-compose file's entrypoint
+    sed -i  "s/tail -f \/dev\/null/\/opt\/docker\/bin\/entrypoint.sh/g" /etc/welder-docker-compose.yaml
```

Comment:
sed is tested against a prod instance

Reply:
Looks like this is Dataproc `else` block? Should it be done for GCE and Dataproc?

Reply:
yea, so the condition is to check if `/var/certs/jupyter-server.crt` exists, which will be `false` if the runtime is old (created before the gpu image change)

Reply:
Ahh I see